### DISCUSSION TITLE
How about changing the font-color for better readability?

### DISCUSSION THREAD

**Discussion by lincyEHVG** [stoplightio/elements#2679] [Ideas]
Perhaps it's an idea to change the font-color, see the picture below for reference.

![afbeelding](https://github.com/user-attachments/assets/f5fdd33d-4d69-4636-9ba0-96bcc0cfa340)

**Comment by weynelucas** [comment]
Same problem when I host the doc page from my own domain. Domain stoplight.io is ok.
